src/main/org/testng/eclipse/launch/SuiteSelector2.java

```diff
@@ -30,7 +30,7 @@
   SuiteSelector2(TestNGMainTab callback, Composite comp) {
     super(callback, comp, LaunchType.SUITE, "TestNGMainTab.label.suiteTest",
         "CheckBoxTable.suites.title");
-    setTextEditable(false); // no manual edit
+    setTextEditable(true); // no manual edit
```

Comment:
You should remove the associated comment - which should never have been there in the first place and is now outdated.